mne/viz/epochs.py

```diff
@@ -119,20 +124,38 @@ def plot_epochs_image(epochs, picks=None, sigma=0., vmin=None,
         ax1, ax2 = axes[:2]
         # if axes were passed - we ignore fig param and get figure from axes
         fig = ax1.get_figure()
+        axes = [ax1, ax2]
         if colorbar:
-            ax3 = axes[-1]
-    evoked = epochs.average(picks)
-    data = epochs.get_data()[:, picks, :]
+            axes += axes[-1]
```

Comment:
Don't review the code right now, I've been putting it on hold until we decide on the API.